Lib/uuid.py

```diff
@@ -666,8 +666,8 @@ def _random_getnode():
     # counting from 1 being the least significant bit, or 1<<40.
     #
     # See https://en.wikipedia.org/w/index.php?title=MAC_address&oldid=1128764812#Universal_vs._local_(U/L_bit)
-    import random
-    return random.getrandbits(48) | (1 << 40)
+
+    return (int.from_bytes(__import__('os').urandom(6), 'big')) | (1 << 40)
```

Comment:
```suggestion
    import os
    return int.from_bytes(os.urandom(6)) | (1 << 40)
```

We don't care about endianess, and avoid using `__import__`

Reply:
it seems like we've import os already lol

Reply:
```
return int.from_bytes(os.urandom(6)) | (1 << 40)
```